frontend/src/components/core/Modal.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useRef, useEffect } from 'react';
+import styled from 'styled-components';
+import { GrayTextButton } from './Button';
+
+const Content = styled.div<ContentProps>`
+  position: fixed;
+  
+  top: ${({ fullScreen }) => (fullScreen ? '10%' : '40%')};
+  height: ${({ fullScreen }) => (fullScreen ? '80%' : '150px')};
+  left: ${({ fullScreen }) => (fullScreen ? '20%' : '30%')};
+  width: ${({ fullScreen }) => (fullScreen ? '60%' : '40%')};  
+  min-height: 150px;
+
+  z-index: ${({ show }) => (show ? 2 : -2)};
```

Comment:
possible nit / question: Should we move this `z-index` higher given that the hover tooltip is at 5 (I believe)?

Reply:
I changed this to be 4 - this way it's higher than the tooltip (3) and lower than the copied indicator (5) - yeah now I can definitely see a potential need to flesh out the z-indices haha

Reply:
Ok perfect 👍